src/main/java/seedu/address/model/BorrowerRecords.java

```diff
@@ -147,4 +147,14 @@ public boolean hasDuplicateBorrower(Borrower editedBorrower) {
                         && (current.getPhone().equals(editedBorrower.getPhone())
                         || current.getEmail().equals(editedBorrower.getEmail())));
     }
+
+    /**
+     * Removes a borrower from the borrower records.
+     *
+     * @param borrower Borrower to be removed.
+     */
+    public void removeBorrower(Borrower borrower) {
+        listOfBorrowers.remove(borrower);
+        borrowersMap.remove(borrower);
```

Comment:
I think the argument to be passed into `borrowersMap.remove` is the BorrowerId since that is the key. So should it be `borrowersMap.remove(borrower.getBorrowerId())`

Reply:
okay noted with thanks!